pages/common/tmux.md

```diff
@@ -1,36 +1,37 @@
 # tmux
 
-> Multiplex several virtual consoles.
+> Tmux is a terminal multiplexer. It lets you switch easily between several programs in one terminal, detach them (they keep running in the background) and reattach them to a different terminal.
```

Comment:
Wow, lots of words here. Perhaps we can trim this down a bit? The description is meant to serve as a brief summary.